### PR TITLE
imap-uw: fix build for Linux

### PR DESCRIPTION
**Pull request by shivammathur**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

This PR:
- Adds `krb5` and `linux-pam` to `imap-uw` as dependencies on Linux and sets the make target for Linux.
- Adds license to the `imap-uw` formula.

### CODE REVIEW

File: Formula/imap-uw.rb [hunk 1]
```diff
@@ -22,6 +23,11 @@ class ImapUw < Formula
 
   depends_on "openssl@1.1"
 
+  on_linux do
+    depends_on "krb5"
```
Comment (carlocab) [inline Formula/imap-uw.rb:27]:
```suggestion
  uses_from_macos "krb5"

  on_linux do
```

**Review by carlocab — COMMENTED** [review]
Thanks, @shivammathur. Just one suggestion.

File: Formula/imap-uw.rb (hunk 1 above)
Comment (shivammathur) [reply Formula/imap-uw.rb]: Done

File: Formula/imap-uw.rb [hunk 2]
```diff
@@ -45,7 +52,16 @@ def install
...
+    on_macos do
+      system "make", "oxp"
+    end
+
+    on_linux do
+      # Skip IPv6 warning on Linux as libc should be IPv6 safe.
+      touch "ip6"
+
+      system "make", "ldb"
+    end
```
Comment (carlocab) [inline Formula/imap-uw.rb:64]: Does this work?
```suggestion
    # Skip IPv6 warning on Linux as libc should be IPv6 safe.
    touch "ip6"

    target = "oxp"
    on_linux { target = "ldb" }
    system "make", target
```

Comment (shivammathur) [reply Formula/imap-uw.rb]: Yes, it should work. Applied the suggestions.

**Review by carlocab — APPROVED** [review]
Thanks, @shivammathur.

**Comment by shivammathur** [comment]
@carlocab Can you dispatch a linuxbrew build for this?

**Comment by carlocab** [comment]
I can't, but someone in @Homebrew/linux can help with that.

**Comment by danielnachun** [comment]
https://github.com/Homebrew/homebrew-core/actions/runs/1111511431

**Comment by shivammathur** [comment]
Thanks, @danielnachun

**Comment by shivammathur** [comment]
@danielnachun Please run with `upload=true`.

**Comment by carlocab** [comment]
@danielnachun there is already a bottle in Homebrew/core. @shivammathur is requesting a Linuxbrew/core bottle.

**Pull request closed without merging**